AggregatorAdapters/goblinsax/index.ts

```diff
@@ -0,0 +1,71 @@
+import { bool } from 'aws-sdk/clients/signer'
```

Comment:
is there a way to avoid this package?

Reply:
Yes! Good catch - that definitely shouldn't be there